### DISCUSSION TITLE
Tag-based control on Notes for Visibility in Notes-Views, Preview, Editing and Password Protection

### DISCUSSION THREAD

**Discussion by xthrk** [standardnotes/forum#2886] [Suggestions]
**Describe the solution you'd like**
It would be nice if there was a way to control the visibility of notes in the Notes-View based on tags. To set a per-tag-setting that notes that have a specific tag are not displayed in Notes-View.

**Describe alternatives you've considered**
Alternatively, the "Show preview" setting could be set in the tags. Per tag you could set whether the preview is shown or not. As soon as a tag with this setting is assigned to a note, the preview is not shown.

**Additional context**
Possibly, you could consider setting Editing and Password Protection per tag as well.

**Comment by shompoe** [comment]
You should already be able to create a custom view that would do this by using the smart view feature. 

https://standardnotes.com/help/42/how-do-i-view-a-list-of-untagged-notes-and-create-other-dynamic-filters

https://github.com/standardnotes/forum/issues/262

**Comment by xthrk** [comment]
Thank you shompoe, for your link. I know the Smart View functionality, but that's not what I meant in this case.

I would like to see either a way that by assigning tags to notes, you can hide those notes in the Main View area (the top left area). I imagine the implementation to be similar to the following, ....
* That you have a tag, let's call it "#private" for example.
* There is then the folder "#private". In this folder you see all the notes with that the tag "#private".
* And the tag then has a setting that all notes from that tag are not displayed in the Main Notes View.

One possible use-case: May you want to show someone something you've jotted down in Standard Notes, it can accidentally end up with someone seeing another note or preview of another note that that person shouldn't see. I know there are other ways, such as hiding the List Panel. What I also like is the way to disable the "Show preview" of a note, but this is globally. And hence the alternative idea:

Adding "Prevent editing", "Show preview" and "Password protect" to Tags (maybe with a detailed setting option per view or only for the main views). 
![example](https://user-images.githubusercontent.com/116081825/199045819-c72cae48-b92a-4460-ba47-2dea9f1c9c51.png)
Let's say that this options exists for tags and you have deactivated "Show preview" for a tag, you could use it to control that for all notes to which this tag is linked, the preview is immediately not displayed, regardless of whether you have additionally deactivated "Show preview" on the note itself or not. And maybe you can say, that the preview of these Notes with this tag is then disabled globally in all other views and folders, except for the folder with the tag itself.

After writing down this idea, I thought a lot about it and if it is something, I really need. I've been thinking about the probability of the use case I wrote down above. And maybe this request is just a low prio nice-to-have, but nothing I really need. Maybe others will find this idea helpful? If not, we might as well close this issue.

**Comment by myreli** [comment]
While it's not possible to exclude a tag from the main view, you can now apply specific settings for a single tag:

<img width="489" alt="image" src="https://user-images.githubusercontent.com/17554234/221379129-b4259448-2b77-46ee-b6a9-a22dc860f037.png">
